packages/base-zone/src/is-passable.js

```diff
@@ -1,23 +0,0 @@
-import { passStyleOf } from '@endo/pass-style';
```

Comment:
I don't see the reason to remove this file until we really want to make an API-breaking change (such as when this package is moved to Endo).  Could you instead replace it with:
```js
export { isPassable } from '@endo/pass-style';
```

Reply:
Done, but a bit more verbosely so that importers from here will see it as deprecated with a suggestion to import it from its new home.